_sass/_articles-main.scss

```diff
@@ -16,6 +16,13 @@
       left: 0;
     }
   }
+  article {
+    img {
+      &.inline {
+        max-width: 100%;
+      }
+    }
+  }
```

Comment:
We have a css class that will do this, so this is unnecessary. See my previous post.